tests/conftest.py

```diff
@@ -145,7 +145,7 @@ def ov_config_dir(request):
 
 
 @pytest.fixture(scope="function")
-def create_tmp_venv_with_nncf(install_type, tmp_path, package_type, venv_type):  # pylint:disable=redefined-outer-name
+def tmp_venv_with_nncf(install_type, tmp_path, package_type, venv_type):  # pylint:disable=redefined-outer-name
```

Comment:
Why don't you like the name?

Reply:
The built-in fixtures such as `tmp_path` and `tmp_dir` do not have `create` in them, so I decided to go for consistency here.